src/TobiasOlry/TalklyBundle/DependencyInjection/TobiasOlryTalklyExtension.php

```diff
@@ -15,6 +15,8 @@ public function load(array $config, ContainerBuilder $container)
             $container,
             new FileLocator(__DIR__ . '/../Resources/config')
         );
+
+        $loader->load('security.xml');
```

Comment:
prefix filename with `services-` to prevent confusion with app security.yml|xml?